Upstream: tentative patch to see if this is the bleachbit root startup fix
From 48cf2594ea7d42a6492b7aaaea52e5bee9d3a8d0 Mon Sep 17 00:00:00 2001
From: Ikey Doherty <ikey@solus-project.com>
Date: Tue, 1 May 2018 10:56:01 +0100
Subject: [PATCH] polkit: Ensure the cancellable is truly unowned

Signed-off-by: Ikey Doherty <ikey@solus-project.com>
---
 src/polkit/polkitdialog.vala | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/polkit/polkitdialog.vala b/src/polkit/polkitdialog.vala
index 397579fc..5f677e07 100644
--- a/src/polkit/polkitdialog.vala
+++ b/src/polkit/polkitdialog.vala
@@ -68,7 +68,7 @@ public class AgentDialog : Gtk.Window
     }
 
     /* Manipulate Vala's pointer logic to prevent a copy */
-    public unowned GLib.Cancellable? cancellable { public set ; public get; }
+    public unowned GLib.Cancellable? cancellable { public set ; public unowned get; }
 
     public string cookie { public get; public set; }
 
-- 
2.17.1

